### PR TITLE
Added a tornado-tilestache example

### PR DESCRIPTION
**Pull request by dorukozturk**
@aashish24 PTAL

### CODE REVIEW

**Comment by aashish24** [comment]
great! can you add a readme on how to setup tornado (or a pointer to some document)?

**Comment by dorukozturk** [comment]
@aashish24 I added some instructions. PTAL

**Comment by aashish24** [comment]
thanks @dorukozturk I see the image but I also seeing this error: 

Traceback (most recent call last):
  File "/home/chaudhary/tools/tilestache/src.git/tilestache_dev/local/lib/python2.7/site-packages/tornado/web.py", line 1467, in _execute
    result = method(_self.path_args, *_self.path_kwargs)
  File "server.py", line 76, in get
    layer_name, z, x, y = components
ValueError: need more than 1 value to unpack

File: examples/tornado/server.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import os
+
+from osgeo import gdal
+from osgeo import osr
+import TileStache
```
Comment (aashish24) [inline examples/tornado/server.py]: could you please have a space between these two groups as python convention?

File: examples/tornado/server.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+import tornado.web
+
+
+def mapnik_config(layer_name, file_path, layer_srs):
+    """ Creates a mapnik config file
+    file_path is the absolute path to
+    the geotiff file """
+
+    return """
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (aashish24) [inline examples/tornado/server.py]: can we indent it?

File: examples/tornado/server.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    <Parameter name="file">{}</Parameter>
+    <Parameter name="format">tiff</Parameter>
+    <Parameter name="band">1</Parameter>
+</Datasource>
+</Layer>
+</Map>
+""".format(layer_name, layer_srs, file_path)
+
+def get_config(layer_name, file_path, layer_srs):
+
```
Comment (aashish24) [inline examples/tornado/server.py]: Nitpick: Is this convention to leave a whitespace after the signature? I typically see that if you have documentation.

File: examples/tornado/server.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        raster = gdal.Open(file_path)
+        srs = osr.SpatialReference()
+        srs.ImportFromWkt(raster.GetProjectionRef())
+        layer_srs = srs.ExportToProj4()
+
+        components = [x for x in self.request.path.split("/") if x]
+        layer_name, z, x, y = components
+        config = get_config(layer_name, file_path, layer_srs)
+        layer = config.layers
+        status_code, headers, content = TileStache.requestHandler2(config, path_info)
```
Comment (aashish24) [inline examples/tornado/server.py]: @dorukozturk do we always has to pass config?

**Comment by dorukozturk** [comment]
@aashish24 The browser is sending one extra request when the page is first loaded. 
The request is to get the favicon. I can do a try except block or add a favicon. I will fix this.

**Comment by dorukozturk** [comment]
@aashish24 https://github.com/OpenGeoscience/KTile/pull/4/commits/ab9d89da9ac77207aa3f9e420b45a3502d91f348 Attempts to fix this.

**Comment by aashish24** [comment]
thanks its better but may not be not totally removed. Also, can you address style issues?

I am getting this now: 
(dev)[chaudhary@einstein tornado (tornado-example)]$ python server.py 
WARNING:tornado.access:404 GET /favicon.ico (127.0.0.1) 1.09ms

**Review by aashish24 — CHANGES_REQUESTED** [review]

**Comment by aashish24** [comment]
@dorukozturk I pushed a change, have a look and let me know if you have any suggestions. I will add some docs and bit more optimization and then will call it done.

**Pull request merged**